src/applications/simple-forms/40-0247/pages/additionalCertificatesYesNo.js

```diff
@@ -0,0 +1,31 @@
+// can't use v3 yesNoUI because it doesn't support JSX ui:title
+/* @type {PageSchema} */
+export default {
+  uiSchema: {
+    additionalCertificates: {
```

Comment:
can you use the web component yes no here? you can see yesNoPattern for example. either you can use yesNoUI/yesNoSchema directly, or include the `ui:webComponentField: YesNoField`

Reply:
Refactored to use v3 yesNoUI & yesNoSchema, but how do we customize the error-messages?

Reply:
Refactored to use v3 yesNoUI & yesNoSchema, but can we customize error-message?